terraform/lib/dependabot/terraform/requirement.rb

```diff
@@ -7,6 +7,13 @@
 module Dependabot
   module Terraform
     class Requirement < Gem::Requirement
+      # Override regex PATTERN from Gem::Requirement to add support for the
+      # optional 'v' prefix to release tag names, which Terraform supports.
+      # https://www.terraform.io/docs/registry/modules/publish.html#requirements
+      quoted = OPS.keys.map { |k| Regexp.quote k }.join "|"
+      PATTERN_RAW = "\\s*(#{quoted})?\\s*v?(#{Gem::Version::VERSION_PATTERN})\\s*"
```

Comment:
```suggestion
      OPERATORS = OPS.keys.map { |k|ey Regexp.quote(key) }.join("|").freeze
      PATTERN_RAW = "\\s*(#{OPERATORS})?\\s*v?(#{Gem::Version::VERSION_PATTERN})\\s*"
```

Reply:
updated this with commit 99a4d42, because typo in `|k|ey`

Reply:
Ah yeah my bad!